hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/validation/country/EidasAttributeStatementAssertionValidator.java

```diff
@@ -106,4 +108,10 @@ private void validateAttributes(Assertion assertion) {
             }
         }
     }
+
+    private boolean isExpectedQName(QName expectedQName, QName attributeValueQName) {
+        return attributeValueQName != null
+                && expectedQName.getPrefix().equals(attributeValueQName.getPrefix())
+                && expectedQName.getLocalPart().equals(attributeValueQName.getLocalPart());
+    }
```

Comment:
This doesn't seem right to me. In this implementation we're not checking the attribute namespace, therefore, we're not checking that the attribute complies with the eIDAS schema.
I believe it's a standard XML practice to define namespaces at the top and then referring to them in children elements - so they can be referred to multiple times without re-defining each time. We should either:
- check if there's a ready XML/OpenSAML method to verify attribute namespaces
- create a map of namespaces in the in `Response` element and then verify the attribute namespace either exists in the attribute or in the map. E.g. for `xsi:type="eidas-natural:CurrentFamilyNameType"` the map should have an entry for key `eidas-natural`

I would check the XML/SAML/eIDAS spec to check where it's allowed to define namespaces to make sure we're looking in the right place.